mreg/validators.py

```diff
@@ -117,9 +117,9 @@ def validate_reverse_zone_name(name):
 
 def validate_mac_address(address):
     """Validates that the mac address is on a valid form."""
-    adr_regex = "^([a-f0-9]{2}:){5}[a-f0-9]{2}$"
+    adr_regex = "^([a-fA-F0-9]{2}[:.-]?){5}[a-fA-F0-9]{2}$"
     validator = RegexValidator(adr_regex,
-                               message="Must be on form: aa:bb:cc:00:11:22")
+                               message="Not a valid MAC address")
```

Comment:
Any reason why you changed the validator? This would make it possible to actually store "aa:AA:aa:bb:ccAA" as a mac-address if created outside the serializer, e.g. signals or scripts.

Reply:
Good point.  I've pushed a v2 that implements this check only in the serializer, and separates the conversion out to a [SerializerMethodField](https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield).